components/Learn/PreviousNextLink/index.module.scss

```diff
@@ -0,0 +1,24 @@
+.prevNextLink {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  list-style: none;
+  margin: 0;
+  padding: 5rem 0 0 0;
+
+  a {
+    align-items: center;
+    color: var(--black7) !important;
```

Comment:
Do we need the `!important` statements here?

Reply:
I remember on nodejs.dev we needed because of global styles... Let's keep for now.